test/acceptance/test-endpoint.js

```diff
@@ -124,10 +127,10 @@ describe('NgFullstack:endpoint', () => {
         'server/api/todo2/model/endpmodel.go',
         'server/api/todo2/routes/endproute.go',
 
-        'server/api/todo2/controller/endpcontroller_test.go',
-        'server/api/todo2/routes/endproute_test.go',
-        'server/api/todo2/dao/endpdao_test.go',
-        'server/api/todo2/model/endpmodel_test.go'
+        'server/api/todo2/controller/endpcontroller.spec.go',
```

Comment:
One thing I overlooked is this suffix change. Go needs the `_test.go`. Sorry for not letting you know before.